### PR TITLE
 Project url changed

### PR DESCRIPTION
**Pull request by adrien5054**
I changed the project url with the one you gave me

### CODE REVIEW

**Review by drjova — APPROVED** [review]

**Review by drjova — CHANGES_REQUESTED** [review]
We don't need the `colums-to-ignore` :)

**Comment by adrien5054** [comment]
Ok I remove it

**Comment by adrien5054** [comment]
I removed this line

File: .github/workflows/move-inprogress-to-review.yml [hunk 1]
```diff
@@ -15,4 +15,4 @@ jobs:
           project-url: https://github.com/orgs/cern-sis/projects/1
           column-name: "Review"
           label-name: "ready-to-review"
-          colums-to-ignore: "On hold, Todo, QA, Ready to Deploy, PROD, Done"
\ No newline at end of file
+
```
Comment (drjova) [inline .github/workflows/move-inprogress-to-review.yml:18]: could you please also add a new line at the end of the file? You can configure your editor to do that by default.

**Comment by adrien5054** [comment]
Sorry but I don't understand what kind of line ? Just an empty one ?

**Comment by drjova** [comment]
This red icon means that we don't have a new line at the end of the file.

Screenshot 2023-06-02 at 13 41 16.

> Adding a newline character at the end of your files is a good practice due to a historical requirement in UNIX systems. The POSIX definition of a line is "a sequence of zero or more non-newline characters followed by a newline character". So if a line doesn't end with a newline character, it's not considered a complete line.

In Visual Studio Code (VS Code), you can ensure that a newline is added to the end of files automatically. To do this, follow these steps:

- Open your settings. You can do this by going to File > Preferences > Settings, or by using the shortcut Ctrl + , (comma).
- Search for "insert final newline".
- Check the box that says "Editor: Insert Final Newline". This ensures that a newline is inserted at the end of the file when it is saved.
- You can also configure VS Code to remove any extra newlines at the end of the file, so that there's always exactly one. To do this, search for "trim final newlines" in the settings and check the box that says "Editor: Trim Final Newlines".

**Comment by adrien5054** [comment]
Is it ok like this ?

**Comment by drjova** [comment]
Unfortunately still has the same issue

**Comment by adrien5054** [comment]
I think it's ok now

**Comment by drjova** [comment]
Great, could you please also check the guidelines [here](https://scientific-information-service.web.cern.ch/SIS%20Tools%20%26%20Services/Software%20Development/Contribute/) for the commit messages?

**Comment by drjova** [comment]
Also, please squash your commit messages into one

**Comment by adrien5054** [comment]
How can I do that ? I tried many ways but unfortunately I failed

**Comment by adrien5054** [comment]
I tried to squash 3 of the 5 commits together to see what would happened but instead it create 3 other commits

**Comment by drjova** [comment]
You can try the following guide: https://gist.github.com/lpranam/4ae996b0a4bc37448dc80356efbca7fa

**Pull request closed without merging**